src/praisonai-deploy/praisonai_deploy/scheduler/deployment.py

```
@@ -0,0 +1,224 @@
+                    self.provider = self.provider if hasattr(self, 'provider') else 'gcp'
+                    self.json = False
+                    self.background = False
+            
+            # Create args with the provider from the scheduler
+            deploy_args = DeployArgs()
+            deploy_args.provider = self.provider
+            
+            handler.handle_deploy(deploy_args)
+            return True
+            
+        except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)